KPP/fullchem/fullchem.kpp

```diff
@@ -17,6 +17,7 @@ LCO : CO;
 PSO4 : SO4;
 LCH4 : CH4;
 PH2O2 : H2O2;
+PRO2: A3O2 + ATO2 + B3O2 + BRO2 + ETO2 + HO2 + LIMO2 + MO2 + PIO2 + PO2 + PRN1 + R4N1 + R4O2 + TRO2 + XRO2 + IHOO1 + IHOO4 + ICHOO + IHPOO1 + IHPOO2 + IHPOO3 + IEPOXAOO + IEPOXBOO + C4HVP1 + C4HVP2 + HPALD1OO + HPALD2OO + ISOPNOO1 + ISOPNOO2 + INO2D + INO2B + IDHNBOO + IDHNDOO1 + IDHNDOO2 + IHPNBOO + IHPNDOO + ICNOO + IDNOO;
```

Comment:
Looks good.  Will also have to add this to the `KPP/custom/custom.kpp` fille as well.